### ISSUE TITLE
Create support for the Admin plugin

### ISSUE THREAD

**Issue by justinhartman** [justinhartman/html5up-astral#2]
The first version will only provide support for an installable theme. Once this is done I will look at how to support the admin plugin. I haven't even used the Admin plugin before, hence the separate ticket for this.

**Comment by 205er** [comment]
Hi,
Do you have plans to work on this issue?

Best regards
